packages/bff/src/config/config.ts

```diff
@@ -5,6 +5,7 @@ import {
   FileManagerConfig,
   TokenGenerationConfig,
 } from "pagopa-interop-commons";
+import { SessionTokenGenerationConfig } from "../../../commons/dist/config/sessionTokenGenerationConfig.js";
```

Comment:
I guess autoimport 😄 
```suggestion
import { SessionTokenGenerationConfig } from "pagopa-interop-commons";
```